### PR TITLE
hypervisor: export SerializeParams and DeserializeParams

### PR DESCRIPTION
**Pull request by wcwxyz**
Add an exported function DeserializeParams(), and export
SerializeParams().

This patch is laying the ground work for [https://github.com/clearcontainers/runtime/pull/369](url)

### CODE REVIEW

File: hypervisor.go [hunk 1]
```diff
@@ -201,6 +203,25 @@ func serializeParams(params []Param, delim string) []string {
...
+	for _, param := range parameters {
+		if param == "" {
+			continue
+		}
+		p := strings.SplitN(param, "=", 2)
+		if len(p) == 2 {
+			params = append(params, Param{Key: p[0], Value: p[1]})
+		} else if len(p) == 1 {
+			params = append(params, Param{Key: p[0], Value: ""})
+		}
```
Comment (sboeuf) [inline hypervisor.go]: You should put an else statement giving us some logs:
```go
virtLog.Warnf("Invalid parameter %s", param)
```

Comment (wcwxyz) [inline hypervisor.go]: Thanks for review.

After strings.SplitN(param, "=", 2), len(p) is either 1 or 2. So another else statement seems like unnecessary. Correct me if I'm wrong.

Comment (sboeuf) [inline hypervisor.go]: Yes you're right, then in that case, use:
```go
else
```
for the second case instead of
```go
else if len(p) == 1
```

Comment (wcwxyz) [inline hypervisor.go]: ok, I'll update.

**Comment by sboeuf** [comment]
@wcwxyz Nice thanks for the changes. The PR is ready to be merged after we got confirmation from the CI.
LGTM

**Pull request merged**